demo-server.html

```diff
@@ -7,300 +7,425 @@
+                <p>
+                    If you are able to log into the Demo Server using the
+                    OMERO.web client at <a class="reference external"
+                    href="http://demo.openmicroscopy.org"
+                    target="_blank">https://demo.openmicroscopy.org</a>, but
+                    the OMERO.insight client won't connect, it's likely that
+                    your institution is blocking the network ports
+                    OMERO.insight needs to communicate over.
+                </p>
 
-<div class="line-block">
-<div class="line"><br /></div> <!-- end #line -->
-</div> <!-- end #line-block -->
+                <p>
+                    Please refer your IT department to the following
+                    documentation for the information they need to fix the
+                    issue for you:
+                </p>
 
-<div class="line-block">
-<div class="line"><br /></div> <!-- end #line -->
-</div> <!-- end #line-block -->
+                <p class="indent3"><a class="reference external"
+href="http://www.openmicroscopy.org/site/support/omero/sysadmins/server-security.html"
+target="_blank">http://www.openmicroscopy.org/site/support/omero/sysadmins/server-security.html</a></p>
 
-<div class="page-end">
-	<p>All Tutorial Material is available on line at: <a href="http://help.openmicroscopy.org">help.openmicroscopy.org</a></p>
-	<p>The Main OME website is at: <a href="http://www.openmicroscopy.org">www.openmicroscopy.org</a></p>
-</div> <!-- end page-end -->
+                <div class="line-block">
+                    <div class="line"><br /></div> <!-- end #line -->
+                </div> <!-- end #line-block -->
+
+            </li>
+
+        </ol>
+
+        <h3>Contact us if you have any problems</h3>
+
+        <p>
+            If you have any problems, questions, suggestions, comments or
+            proposals, please take advantage of the OME community&#8217;s
+            expertise and join our forums or mailing lists:
+        </p>
+        <p class="indent3"><a class="reference external"
+            href="http://www.openmicroscopy.org/community/"
+            target="_blank">Forums</a></p>
+        <p class="indent3"><a class="reference external"
+            href="http://www.openmicroscopy.org/site/community/mailing-lists/"
+            target="_blank">Mailing Lists</a></p>
+
+        <div class="line-block">
+            <div class="line"><br /></div> <!-- end #line -->
+        </div> <!-- end #line-block -->
+
+        <p class="first">
+            For more information on OME and OMERO visit our main
+            site:
+        </p>
+
+        <p class="indent3"><a class="reference external"
+            href="http://www.openmicroscopy.org/site" target="_blank">Open
+            Microscopy Environment</a></p>
+
+        <p class="top_link"><a href="#top"><img src="images/back_to_top.png"
+            alt="Image for link to top of page" style="float:
+            right;"/></a></p>
```

Comment:
A few more ```<p>``` could be fixed here and elsewhere, but I don't think we need to fix every single one to merge this. It's certainly much improved and we can continue to tweak the minor fixes as we go...